tests/Command/PullRequest/PullRequestCreateCommandTest.php

```diff
@@ -132,7 +97,7 @@ public function opens_pull_request_to_a_specific_organization_or_username($args)
         $tester->execute($args, ['interactive' => false]);
 
         $res = trim($tester->getDisplay(true));
-        $this->assertContains('Making PR from '.$args['--source-org'].':issue-145 to gushphp:master', $res);
+        $this->assertContains('Opened pull request', $res);
```

Comment:
https://github.com/sstok/gush/blob/418-pull-reques-create-remove-issue-option/src/Command/PullRequest/PullRequestCreateCommand.php#L152-L158

https://github.com/sstok/gush/blob/418-pull-reques-create-remove-issue-option/tests/Command/PullRequest/PullRequestCreateCommandTest.php#L50-L51

 ;) its already shown in normal mode.